src/dataMigration.js

```diff
@@ -73,6 +73,27 @@ const dataMigrations = [
           }
         });
       });
+
+      const nameResults = database.objects('Name').filtered('id = $0',
+                                settings.get(SETTINGS_KEYS.SUPPLYING_STORE_NAME_ID));
+      if (nameResults.length < 1) throw new Error('Supplying Store Name ID missing from settings');
+      const mainSupplyingStoreName = nameResults[0];
+
+      const requisitions = database.objects('Requisition')
+                                   .filtered('otherStoreName = Null').snapshot();
+
+      database.write(() => {
+        requisitions.forEach(requisition => {
+          if (!requisition.isRequest) return;
```

Comment:
I am pretty sure realm doesn't store getters + functions from datatype in app data. It will construct Requisition data type as it load realm records. So yeah it will have the getter without migration.

In V3 mSupply api four types of requisition, finalised(fn) and suggested(sg) for both isReponse and isRequest, so either isResponse or isRequest have same 'safeness' level